src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -279,27 +289,41 @@ private LineColumn FormatTrivia<T>(Formatter<T> formatter, WhitespaceAppender<T>
             var implicitLineBreak = false;
 
             var list = new TriviaList(this.Token1.TrailingTrivia, this.Token2.LeadingTrivia);
+            var i = -1;
```

Comment:
'i' doens't really mean anything.  can you explain what it's for?  if ti's an index, please change the foreach loop to a standard for loop.

Reply:
Fixed